src/lib/stitching/gravity/stitching.ts

```diff
@@ -8,12 +9,35 @@ export const gravityStitchingEnvironment = (localSchema: GraphQLSchema) => {
       extend type ViewingRoomArtwork {
         artwork: Artwork
       }
-    `,
 
+      extend type ViewingRoom {
+        artworks: ArtworkConnection
```

Comment:
A connection usually accepts pagination arguments. Like `artworks(first: Int, after: Int)`. Also, should rename to `artworksConnection`.

Reply:
- Thanks for teaching me about the pagination arguments. I've included those.
- I can't rename it to `artworksConnection` at the moment because the `ViewingRoom` type already has a field by that name. I will open a PR in Gravity to refactor that field to be an array of all `ViewingRoomArtwork`s that belong to a viewing room.

Reply:
[Here](https://github.com/artsy/gravity/pull/12951) is the Gravity PR.

I added a new field `viewingRoomArtworks` instead of refactoring the existing `artworksConnection` to avoid breaking the existing schema. My plan is to open a separate set of PRs to Gravity and Metaphysics to remove the existing `artworksConnection` field and rename `viewingRoomArtworks` once it is wired-up correctly and I've had a chance to coordinate with @mdole.

Please let me know if you think I'm being overly cautious with this refactor.